libcnb/Cargo.toml

```diff
@@ -12,13 +12,11 @@ readme = "../README.md"
 include = ["src/**/*", "../LICENSE", "../README.md"]
 
 [dependencies]
-lazy_static = "^1.4.0"
-regex = "^1.5.4"
-semver = { version = "^1.0.4", features = ["serde"] }
 serde = { version = "^1.0.126", features = ["derive"] }
 thiserror = "^1.0.26"
 toml = "^0.5.8"
 anyhow = { version = "^1.0.42", optional = true }
+libcnb-data = { path = "../libcnb-data" }
```

Comment:
Does there need to be a version number here in addition to the `path`, so this works when published to crates.io? 
https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-path-dependencies

> However, crates that use dependencies specified with only a path are not permitted on crates.io. 

The solution appears to be to set both:
https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#multiple-locations

Reply:
Good catch! I wasn't aware of that restriction when publishing to crates.io. There seems to be an effort going on to simplify use-cases like ours, but it hasn't landed yet: https://github.com/rust-lang/cargo/issues/3931